### ISSUE TITLE
How can I use this without a remote repository?

### ISSUE THREAD

**Issue by NilsEnevoldsen** [rtyley/bfg-repo-cleaner#70]
I don't have a remote repository. I am working entirely locally. This is what I have done so far:

    git clone --mirror ~/path/to/project/
    java -jar ~/Desktop/bfg-1.12.0.jar --strip-blobs-bigger-than 100M project.git
    git reflog expire --expire=now --all && git gc --prune=now --aggressive
All these commands seemed successful, but I'm not sure what to do next. I think `git push` doesn't make sense.

**Comment by NilsEnevoldsen** [comment]
I tried backing up and removing my old .git folder, and then

    git --git-dir=/path/to/file-compacted-by-bfg.git --work-tree=. init && echo "gitdir: /file-compacted-by-bfg.git"

And added a .git file to the repo directory (unsure of terminology) containing

    gitdir: /users/nilswork/old.git

When I did that, my history looked OK (e.g. large files had been removed), but I also appeared to have uncommitted changes for the removal of every file in the project (even though they were still there on my disk). I was too scared to "ignore all changes", so instead I switched back to my uncompacted .git folder.
